src/frontend/apps/desk/src/features/teams/member-management/components/MemberGrid.tsx

```diff
@@ -104,21 +111,51 @@ export const MemberGrid = ({ team, currentRole }: MemberGridProps) => {
 
   return (
     <>
-      {currentRole !== Role.MEMBER && (
-        <Box $margin={{ all: 'big', bottom: 'small' }} $align="flex-end">
-          <Button
-            aria-label={t('Add members to the team')}
-            style={{
-              width: 'fit-content',
-              minWidth: '8rem',
-              justifyContent: 'center',
-            }}
-            onClick={() => setIsModalMemberOpen(true)}
-          >
-            {t('Add a member')}
-          </Button>
+      <Box
+        $display="flex"
+        $direction="column"
+        $margin={{ all: 'big', bottom: 'small' }}
+        $position="relative"
+      >
```

Comment:
To give more place to the datagrid.
```suggestion
      <Box $margin={{ horizontal: 'big', bottom: 'small' }}>
```